instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -214,8 +214,8 @@ def uninstrument_connection(connection):
     Returns:
         An uninstrumented connection.
     """
-    if isinstance(connection, wrapt.ObjectProxy):
-        return connection.__wrapped__
+    if isinstance(connection, _TracedConnectionProxy):
```

Comment:
Any reason why we need to use `_TracedConnectionProxy` for type check and not just `TracedConnectionProxy`?

Reply:
Yes, we need to check the type of the connection and the `TracedConnectionProxy` is inside a function making it inaccessible for type comparison.